model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -108,12 +108,12 @@ private fun applyCurationToPackage(targetPackage: CuratedPackage, curation: Pack
     val vcs = curation.vcs?.let {
         // Curation data for VCS information is handled specially so we can curate only individual properties.
         VcsInfo(
-            type = it.type ?: base.vcs.type,
-            url = it.url ?: base.vcs.url,
-            revision = it.revision ?: base.vcs.revision,
-            path = it.path ?: base.vcs.path
-        )
-    } ?: base.vcs
+            type = it.type ?: base.vcsProcessed.type,
+            url = it.url ?: base.vcsProcessed.url,
+            revision = it.revision ?: base.vcsProcessed.revision,
+            path = it.path ?: base.vcsProcessed.path
+        ).normalize()
+    } ?: base.vcsProcessed
```

Comment:
I think in line 132 below `vcs` now needs to stay unchanged and `vcsProcessed` needs to be set instead. This way `vcs` still represents the unmodified metadata of the package.